search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -43,7 +43,7 @@
                        :sensor :data-center-h :measurement}
      :always-case-sensitive #{:echo-collection-id}
      :disallow-pattern #{:echo-collection-id}
-     :allow-or #{:attribute :science-keywords :science-keywords-h}}))
+     :allow-or #{:attribute :science-keywords :science-keywords-h :varialbes-h}}))
```

Comment:
It turns out this line of code is obsolete. The options check has been moved to `valid-parameter-options`. That's why all tests (including the option or test) passed. I have removed this line.